conf/modules/cv_colorfilter.xml

```diff
@@ -27,7 +27,7 @@
   </header>
 
   <init fun="colorfilter_init()"/>
-  <makefile target="ap">
+  <makefile>
```

Comment:
Checked it, you're right. I hadn't realized there were other targets than ap and nps.